hybridauth/library/src/Provider/Facebook.php

```diff
@@ -47,7 +47,7 @@ class Facebook extends OAuth2
     /**
      * {@inheritdoc}
      */
-    protected $apiBaseUrl = 'https://graph.facebook.com/v2.8/';
+    protected $apiBaseUrl = 'https://graph.facebook.com/v2.12/';
```

Comment:
You must change the last active version. The V3.3.

Reply:
Code is updated to use Facebook API Version 3.3